presto-product-tests/src/main/java/com/facebook/presto/tests/ImmutableLdapObjectDefinitions.java

```diff
@@ -0,0 +1,114 @@
+                    .setAttributes(ImmutableMap.of(
+                            "cn", "ChildGroup",
+                            "member", "uid=ChildGroupUser,ou=Asia,dc=presto,dc=testldap,dc=com"))
+                    .addObjectClasses(Arrays.asList("groupOfNames"))
+                    .build();
+
+    public static final LdapObjectDefinition DEFAULT_GROUP_USER =
+            LdapObjectDefinition.builder("DefaultGroupUser")
+                    .setDistinguishedName("uid=defaultgroupuser,ou=Asia,dc=presto,dc=testldap,dc=com")
+                    .setAttributes(ImmutableMap.of(
+                            "cn", "DefaultGroupUser",
+                            "sn", "DefaultGroupUser",
+                            "userPassword", "LDAPPass123",
+                            "memberOf", "cn=DefaultGroup,ou=America,dc=presto,dc=testldap,dc=com"
+                    ))
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
+
+    public static final LdapObjectDefinition PARENT_GROUP_USER =
+            LdapObjectDefinition.builder("ParentGroupUser")
+                    .setDistinguishedName("uid=parentgroupuser,ou=Asia,dc=presto,dc=testldap,dc=com")
+                    .setAttributes(ImmutableMap.of(
+                            "cn", "ParentGroupUser",
+                            "sn", "ParentGroupUser",
+                            "userPassword", "LDAPPass123",
+                            "memberOf", "cn=ParentGroup,ou=America,dc=presto,dc=testldap,dc=com"
+                    ))
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
+
+    public static final LdapObjectDefinition CHILD_GROUP_USER =
+            LdapObjectDefinition.builder("ChildGroupUser")
+                    .setDistinguishedName("uid=childgroupuser,ou=Asia,dc=presto,dc=testldap,dc=com")
+                    .setAttributes(ImmutableMap.of(
+                            "cn", "ChildGroupUser",
+                            "sn", "ChildGroupUser",
+                            "userPassword", "LDAPPass123",
+                            "memberOf", "cn=ChildGroup,ou=America,dc=presto,dc=testldap,dc=com"
+                    ))
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
+
+    public static final LdapObjectDefinition ORPHAN_USER =
+            LdapObjectDefinition.builder("OrphanUser")
+                    .setDistinguishedName("uid=orphanuser,ou=Asia,dc=presto,dc=testldap,dc=com")
+                    .setAttributes(ImmutableMap.of(
+                            "cn", "OrphanUser",
+                            "sn", "OrphanUser",
+                            "userPassword", "LDAPPass123"
+                    ))
+                    .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
+                    .build();
+}
```

Comment:
As discussed, kindly check if we can put the passwords, domain,etc in some variables.